src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -13,77 +13,47 @@
  * Used to handle the data validation aspect e.g. validate emails, names, etc.
  */
 public class FieldValidator {
-    // ////////////////////////////////////////////////////////////////////////
-    // ////////////////// Generic types ///////////////////////////////////////
-    // ////////////////////////////////////////////////////////////////////////
+
+    /////////////////
+    // FIELD TYPES //
+    /////////////////
+
+    // name-related
+    public static final String PERSON_NAME_FIELD_NAME = "a person name";
```

Comment:
@wkurniawan07 Fixing this will require modifying some of the final error messages, and I'm actually leaving wording changes for another issue. (I forgot to mention this in the pull request description.)